COMETwebapp.Tests/Components/ReferenceData/MeasurementScales/MappingToReferenceScalesTableTestFixture.cs

```diff
@@ -0,0 +1,133 @@
+// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="MappingToReferenceScalesTableTestFixture.cs" company="RHEA System S.A.">
+//    Copyright (c) 2023-2024 RHEA System S.A.
```

Comment:
Please update Copyright. Ask @samatrhea for the correct one. Update the template directly